pkg/kubelet/rkt/rkt.go

```diff
@@ -62,11 +62,11 @@ const (
 	RktType                      = "rkt"
 	DefaultRktAPIServiceEndpoint = "localhost:15441"
 
-	minimumAppcVersion       = "0.7.4"
-	minimumRktBinVersion     = "1.2.1"
-	recommendedRktBinVersion = "1.2.1"
+	minimumAppcVersion       = "0.8.1"
+	minimumRktBinVersion     = "1.6.0"
+	recommendedRktBinVersion = "1.6.0"
 	minimumRktApiVersion     = "1.0.0-alpha"
-	minimumSystemdVersion    = "219"
+	minimumSystemdVersion    = "229"
```

Comment:
@tmrts Is this required? I think this is the systemd on the host, so 219 is ok. cc @sjpotter